src/uct/ib/base/ib_iface.c

```diff
@@ -56,6 +56,11 @@ ucs_config_field_t uct_ib_iface_config_table[] = {
   {"", "", NULL,
    ucs_offsetof(uct_ib_iface_config_t, super), UCS_CONFIG_TYPE_TABLE(uct_iface_config_table)},
 
+  {"SEG_SIZE", "8192",
+   "Maximal size of copy-out sends. The transport is allowed to support any size\n"
+   "up to this limit, the actual size can be lower due to transport constraints.",
```

Comment:
this documentation as copied from base is irrelevant.
"Size of bounce buffers used for post_send and post_recv"

Reply:
fixed